diagnostic.md

```diff
@@ -157,7 +192,9 @@ div {
 ```
 
 <!-- your answer starts here -->
-
+divOne will float to the left corner of the screen and take along divTwo and
+divThree to its right.  divFour will stay inline below divOne because it was
+cleared of the float.
 <!-- your answer ends here -->
 
 ## Question 7
```

Comment:
137.5 px - Reason being that since it's min-width, the media query will have .stretch be 25% and at 550px width, 25% is 137.5 px (which actually comes out to 137px).